middleware/parse/parse_test.go

```diff
@@ -146,6 +183,72 @@ func TestParseMiddleware(t *testing.T) {
 		assert.Equal(t, http.StatusOK, result.StatusCode)
 	})
 
+	t.Run("Invalid Multipart", func(t *testing.T) {
+		// Write empty body, which is not allowed for content multipart.
+		writer := multipart.NewWriter(nil)
+
+		request := testutil.NewTestRequest(http.MethodPost, "/parse", nil)
+		request.Lang = server.Lang.GetDefault()
+		request.Header().Set("Content-Type", writer.FormDataContentType())
+
+		result := server.TestMiddleware(&Middleware{}, request, func(resp *goyave.Response, _ *goyave.Request) {
+			resp.Status(http.StatusBadRequest)
+		})
+
+		assert.NoError(t, result.Body.Close())
+		assert.Equal(t, http.StatusBadRequest, result.StatusCode)
+		assert.NotNil(t, request.Extra[goyave.ExtraParseError{}])
```

Comment:
This could provide a better error message when it occurs.


```suggestion
		assert.Contains(t, request.Extra, goyave.ExtraParseError{})
```

Reply:
Using require should be preferred

Reply:
`NotNil` is fine for this case.